dev/router.php

```diff
@@ -14,4 +14,4 @@
     header($header->toString());
 }
 
-echo $response->getBody();
+echo $response->getContent();
```

Comment:
@bbatsche - This was required and I don't know the difference between getBody and getContent. My assumption was that content remains encoded, and puts that responsibility on the browser whereas body was decoded already, causing an issue when the browser attempts to double decode. Any issue with this?